silo/tests/test_customformview.py

```diff
@@ -130,7 +131,7 @@ def test_create_customform_success_superuser(self):
         silo_id = response.data['id']
         silo = Silo.objects.get(pk=silo_id)
         form_name = '{} - {}'.format(data['name'], wflvl1.name)
-        fields.append({'name': 'submission_time', 'type': 'time'})
+        fields = fields + CustomFormViewSet._default_columns
```

Comment:
`fields +=`

Reply:
Something else? I'd prefer to change everything at once.